src/ros_commo.py

```diff
@@ -180,7 +180,7 @@ def gaze_at_point(self, x, y, z):
 	# Turn head towards the given target point.
 	# Coordinates: meters; x==forward, y==to Eva's left.
 	def look_at_point(self, x, y, z):
-		print "look at point: ", x, y, z
+		#print "look at point: ", x, y, z
```

Comment:
why are you commenting this out? These are useful for tracking what the robot should be doing ...

Please have other people review your code before committing changes.